gensrc/thrift/PaloBrokerService.thrift

```diff
@@ -67,6 +67,7 @@ struct TBrokerListResponse {
 struct TBrokerOpenReaderResponse {
     1: required TBrokerOperationStatus opStatus;
     2: optional TBrokerFD fd;
+    3: optional i64 size; //file size
```

Comment:
Do we need this size?

Reply:
When read a parquet file by RPC, libparquet.a must kown the file size so that it can parse file. for example: When it read footer information.